operator/v1/types_csi_cluster_driver.go

```diff
@@ -168,6 +168,20 @@ type AWSCSIDriverConfigSpec struct {
 	// +kubebuilder:validation:Pattern:=`^arn:(aws|aws-cn|aws-us-gov|aws-iso|aws-iso-b|aws-iso-e|aws-iso-f):kms:[a-z0-9-]+:[0-9]{12}:(key|alias)\/.*$`
 	// +optional
 	KMSKeyARN string `json:"kmsKeyARN,omitempty"`
+
+	// efsVolumeMetricsRefreshPeriod specifies the frequency, in minutes, at which volume metrics are refreshed.
+	// This field enables metric collection when set to a value greater than 0.
+	// The default refresh period in the EFS CSI driver is 240 minutes.
+	// +openshift:enable:FeatureGate=AWSEFSDriverVolumeMetrics
+	// +optional
+	EFSVolumeMetricsRefreshPeriod int `json:"efsVolumeMetricsRefreshPeriod,omitempty"`
+
+	// efsVolumeMetricsFSRateLimit defines the rate limit, in goroutines per file system, for processing volume metrics.
+	// Requires efsVolumeMetricsRefreshPeriod to be set to a value greater than 0.
+	// The default rate limit set by the EFS CSI driver is 240 goroutines per file system.
+	// +openshift:enable:FeatureGate=AWSEFSDriverVolumeMetrics
+	// +optional
+	EFSVolumeMetricsFSRateLimit int `json:"efsVolumeMetricsFSRateLimit,omitempty"`
```

Comment:
Would it be better to put these into a struct, so you don't have such long names/duplication?
```
efsVolume:
  metrics:
    rateLimit: ...
    refreshPeriod: ...
```